app/schemas/webhook_schemas.py

```diff
@@ -71,3 +71,7 @@ class ActivityContentEvent(WebhookPayload):
 
 class PublishContentEvent(WebhookPayload):
     publish_id: str
+
+
+class ProfileAnalyticsEvent:
```

Comment:
without this pydantic validation won't happen
```suggestion
class ProfileAnalyticsEvent(BaseModel):
```